examples/example_0.py

```diff
@@ -36,22 +39,24 @@ def example_opt(orca_code, nproc, submit=True):
                     'nproc': nproc,
                 }
             },
-            'input_keywords': ['PBE', 'SV(P)', 'Opt'],
+            'input_keywords': ['PBE', 'def2-SVP', 'Opt'],
```

Comment:
why is this change necessary?

Reply:
Not necessary at all, I think I was just trying to harmonize the used basis set across the files (and `def2-SVP` is a more modern version so it's better to use it as an example I think). (in general there are a bunch of cleanups in these files that I did, and separating them out would be too much extra work for me)

Reply:
Ok. fine to keep then